packages/beauty-portal-theme/src/components/BackToTop/styles.scss

```diff
@@ -1,5 +1,5 @@
 .bp-backToTop {
-  background-color: var(--primary-color);
+  background-color: #2e73e1;
```

Comment:
Is it possible to use `var(--primary-color)` instead?

Reply:
missed that one